LICENCE

```diff
@@ -0,0 +1,8 @@
+Copyright © 2020, PeopleDoc Inc.
```

Comment:
It was "Copyright (C) 2012 Rémy HUBSCHER" originally.
I just discovered copyright and license were modified without @Natim's (original author and copyright holder) consent...
+1 to set it to whatever @Natim wants!

Reply:
if it was done while he was working at novapost/peopledoc, then the copyright should be the company :shrug: